ui/src/pages/QuickSetup/LoginPage.scss

```diff
@@ -0,0 +1,74 @@
+@use 'themes/variables' as *;
```

Comment:
this file is just a copy of the styling code that used to be in `QuickSetup.scss` but I thought it deserved its own file